diff --git a/SConstruct b/SConstruct
index 68a50dd..f9c7af1 100644
--- a/SConstruct
+++ b/SConstruct
@@ -556,6 +556,8 @@ config_options = [
            with a prefix like '/opt/cantera'. 'debian' installs to the stage
            directory in a layout used for generating Debian packages.""",
      defaults.fsLayout, ('standard','compact','debian')),
+    ('cantera_version',
+     'Version of Cantera being built.', '')
 ]
 
 opts.AddVariables(*config_options)
@@ -603,7 +605,6 @@ for arg in ARGUMENTS:
         sys.exit(1)
 
 # Require a StrictVersion-compatible version
-env['cantera_version'] = "2.3.0a3"
 ctversion = StrictVersion(env['cantera_version'])
 # For use where pre-release tags are not permitted (MSI, sonames)
 env['cantera_pure_version'] = '.'.join(str(x) for x in ctversion.version)
